src/k8s-extension/setup.py

```diff
@@ -33,7 +33,7 @@
 # TODO: Add any additional SDK dependencies here
 DEPENDENCIES = []
 
-VERSION = "1.4.0"
+VERSION = "1.4.1"
```

Comment:
Looks like the previous update did not increment the version - fyi @bavneetsingh16 .
@bragi92, pl. change this to 1.4.2, which should have been the version for this update

Reply:
1.4.1 is good, it is not yet released